### ISSUE TITLE
[BUG] Z Probe wizzard restrarting printer.

### ISSUE THREAD

**Issue by th0m4sek** [MarlinFirmware/Marlin#19738]
Board MKS Robin Nano v1.2
I's using TFT_COLOR_UI and bltouch.
Wizard starts and printer gets restarted after homing XYZ.

**Comment by boelle** [comment]
Configurations, please
Please ZIP up your `Configuration.h` and `Configuration_adv.h` files (as
requested in the Issue template) and drop them into your next reply.
We'll check them over and see if anything is amiss.

**Comment by boelle** [comment]
Please test the [`bugfix-2.0.x branch`](https://github.com/MarlinFirmware/Marlin/tree/bugfix-2.0.x) to see where it stands. If the problem has been resolved then we can close this issue. If the issue isn't resolved yet, then we should investigate further.

**Comment by th0m4sek** [comment]
[Marlin.zip](https://github.com/MarlinFirmware/Marlin/files/5380747/Marlin.zip)
I'm on bugfix branch already.

**Comment by rhapsodyv** [comment]
Color UI currently don't support Z Probe Wizzard.
```
Linking .pio/build/mks_robin_nano35/firmware.elf
.pio/build/mks_robin_nano35/src/src/lcd/menu/menu_probe_offset.cpp.o: In function `probe_offset_wizard_menu()':
/Users/victor/Development/Marlin/Marlin/src/lcd/menu/menu_probe_offset.cpp:93: warning: undefined reference to `lcd_moveto(unsigned char, unsigned char)'
.pio/build/mks_robin_nano35/src/src/lcd/menu/menu_probe_offset.cpp.o: In function `lcd_put_u8str(char const*)':
/Users/victor/Development/Marlin/Marlin/src/lcd/menu/../lcdprint.h:161: warning: undefined reference to `lcd_put_u8str_max(char const*, unsigned short)'
```

**Comment by rhapsodyv** [comment]
Fixed here: #19742
